app/src/main/java/com/sunasterisk/rememory/data/source/WorkDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.sunasterisk.rememory.data.source
+
+import com.sunasterisk.rememory.data.model.Work
+import com.sunasterisk.rememory.data.source.base.OnDataLoadedCallback
+
+interface WorkDataSource {
+
+    interface Local {
+        fun getWorks(callback: OnDataLoadedCallback<List<Work>>)
+
+        fun addWork(work: Work, callback: OnDataLoadedCallback<Boolean>)
+
+        fun deleteWork(id: String, callback: OnDataLoadedCallback<Boolean>)
+
+        fun updateWorks(task: Work, callback: OnDataLoadedCallback<Work>)
```

Comment:
```suggestion
        fun updateWork(task: Work, callback: OnDataLoadedCallback<Work>)
```
1 work hay nhiều works em?

Reply:
Em viết lộn

Reply:
Em viết thừa s